README.md

```diff
@@ -69,6 +69,17 @@ Install Kerberos client to enable `skt` authentication to Beaker:
 Install `skt` dependencies following its
 [README.md](https://github.com/RH-FMK/skt).
 
+Run tests
+---------
+
+For running all tests write down:
```

Comment:
"write down" would mean e.g. "write something down on a piece of paper". I.e. the reader is requested to make a note of the following command somewhere else. That will certainly not run the tests :) Personally, I would write something like "To run all tests execute:". This can be fixed in a separate PR.

Reply:
I think the wording here and below was copied over from skt, so it will need to be changed in both :) That said, I agree with the comments.